### ISSUE TITLE
How to use in Vue3?

### ISSUE THREAD

**Issue by gxlinhai** [Maronato/vue-toastification#144]
## Versions

- 1.0
- 2.0

## Describe the bug

This is my code.
`<link rel="stylesheet" href="https://unpkg.com/vue-toastification@2.0.0-beta.7/dist/index.css" />`
`<script src="https://unpkg.com/vue@next"></script> `
`<script src="https://unpkg.com/vue-toastification@2.0.0-beta.7/dist/umd/index.min.js"></script>`
`<div id="app"> <button v-on:click="todo"> Fire </button></div>`
`<script>`
`  var App =  {` 
`     setup() {` 
`        const toast = VueToastification. useToast();` 
`       var todo = ()=>{` 
`         console.log('todo...');` 
`         toast("I'm a toast!");` 
`       }` 
`       return { toast, todo }` 
`     }` 
`   }` 
` Vue.createApp(App).mount('#app');`
`</script>`

## Expected behavior

Instead of breaking the page, Y should happen...

## Steps to reproduce

Reproduction [demo](link-to-demo).
Steps:
1. Install normally
2. Set up plugin
3. ...

**Comment by gxlinhai** [comment]
`<link rel="stylesheet" href="https://unpkg.com/vue-toastification@2.0.0-beta.7/dist/index.css">
<script src="https://unpkg.com/vue@next"></script> 
<script src="https://unpkg.com/vue-toastification@2.0.0-beta.7/dist/umd/index.min.js"></script>

<div id="app">
  <button v-on:click="todo"> Fire </button>
</div>

<script>
  var App =  {
    setup() {
      const toast = VueToastification. useToast();
      var todo = ()=>{
        console.log('todo...');
        toast("I'm a toast!");
      }
      return { toast, todo }
    }
  }
Vue.createApp(App).mount('#app');
</script>`

**Comment by gxlinhai** [comment]
It's not function

**Comment by Maronato** [comment]
Hey @gxlinhai, did you figure it out?

For future reference: 
- Vue Toastification **v1** is for **Vue 2**.
- Vue Toastification **v2+** is for **Vue 3**.

As of right now, **v2** is in beta and is available through the `next` branch on NPM and here on Github.

You can check out the docs by clicking the link mentioned on the readme:
![CleanShot 2020-11-02 at 10 03 58@2x](https://user-images.githubusercontent.com/19658460/97871523-6611b800-1cf3-11eb-9463-780d022f2526.png)


tl;dr:
For Vue 3, do:
```
npm install vue-toastification@^2
yarn add vue-toastification@^2
```
